lib/init/grass.py

```diff
@@ -2110,11 +2111,11 @@ def clean_temp():
     nul.close()
 
 
-def clean_all():
+def clean_all(*, start_time):
```

Comment:
I understand you like this new syntax `(*, keyword)`, but is it really needed here? Golden rule: Keep code sufficiently simple.

Reply:
I don't think it is necessary in this particular function or PR, but it contributes to both documentation and enforcement of correct use and it seems as something which is becoming standard Python now (also judging from introduction of the `/` to mark positional-only).

Python is using it (at least [os.path in 3.10](https://docs.python.org/3.10/library/os.path.html#os.path.realpath): `os.path.realpath(path, *, strict=False)`) and [scipy](
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.sampling.DiscreteAliasUrn.html#scipy.stats.sampling.DiscreteAliasUrn) does too: `scipy.stats.sampling.DiscreteAliasUrn(dist, *, domain=None, urn_factor=1, random_state=None)`.

As documentation, it is not needed for running the code, but it seems to be increasingly used for documenting Python.

As an enforcement of using `start_time` as only a keyword argument, `*` is the minimal code needed to specify that rule.

`*` says "I may add more parameters in the future and I don't know how I will order them." This is quite fitting here and many other places in the Python API.